notifications/management/commands/send_notifications.py

```diff
@@ -171,46 +198,31 @@ def handle(self, *args, **options):
             dthandler = lambda x: x.isoformat() if isinstance(x, date) else None
             self.stdout.write(json.dumps(output, default=dthandler))
 
-
-    def find_bill_action_updates(self, bill_ids):
-
-        new_actions = '''
-            SELECT DISTINCT ON (bill.ocd_id)
-              bill.slug AS bill_slug,
-              bill.identifier AS bill_identifier,
-              bill.description AS bill_description,
-              action.description AS action_description,
-              action.date AS action_date
-            FROM new_action AS new
-            JOIN councilmatic_core_bill AS bill
-              ON new.bill_id = bill.ocd_id
-            JOIN councilmatic_core_action AS action
-              ON new.bill_id = action.bill_id
-            WHERE new.bill_id IN %s
-        '''
-
-        cursor = connection.cursor()
-        cursor.execute(new_actions, [tuple(bill_ids)])
+    def find_bill_action_updates(self, bill_ids, minutes=15):
+        new_actions = councilmatic_models.BillAction.objects.filter(
+            bill__id__in=bill_ids,
+            date_dt__gte=self.get_threshold(minutes)
```

Comment:
hmm, this column refers to the date of the action itself, not necessarily when it was added to the database. my first instinct was to query through the updated at on the bill, however that can be toggled for any bill update, not just new actions. we used to create a new table by left joining incoming objects with the existing table to separate them into new and updated objects. i don't know that we have a good way of knowing whether a bill action (or other event- or bill-related object without its own updated at timestamp) is "new" in the refactor... @fgregg, do you have any thoughts here?